mods/ra/rules/misc.yaml

```diff
@@ -388,3 +390,4 @@ CTFLAG:
 	Invulnerable:
 	-Selectable:
 	-TargetableBuilding:
+
```

Comment:
Actually not having it causes constant noise by the upgrade utility, which tries to add it every time I run it. Same applies for a bunch of other files. For some reason it wants the files to end with 2 empty lines.